reciprocalspaceship/dataset.py

```diff
@@ -597,8 +597,7 @@ def apply_symop(self, symop, inplace=False):
 
         # Shift phases according to symop
         for key in dataset.get_phase_keys():
-            dataset[key] += np.rad2deg(phase_shifts)
-            dataset[key] *= phic
+            dataset[key] = phic * (dataset[key] - np.rad2deg(phase_shifts))
             dataset[key] = utils.canonicalize_phases(dataset[key], deg=True)
         for key in dataset.get_complex_keys():
             dataset[key] *= np.exp(1j*phase_shifts)
```

Comment:
Don't you need to change the sign of this complex rotation too?

Reply:
Fixed in latest update -- the phase shifts have now been flipped for complex structure factors as well